code/modules/organs/organ_external.dm

```diff
@@ -911,351 +801,209 @@ Note that amputating the affected organ does in fact remove the infection from t
 		H.drop_item()
 	W.loc = owner
 
+/obj/item/organ/external/removed(var/mob/living/user, var/ignore_children)
+
+	if(!owner)
+		return
+	var/is_robotic = status & ORGAN_ROBOT
+	var/mob/living/carbon/human/victim = owner
+
+	..()
+
+	status |= ORGAN_DESTROYED
+	victim.bad_external_organs -= src
+
+	for(var/implant in implants) //todo: check if this can be left alone
+		del(implant)
+
+	// Attached organs also fly off.
+	if(!ignore_children)
+		for(var/obj/item/organ/external/O in children)
+			O.removed()
+			if(O)
+				O.loc = src
+
+	// Grab all the internal giblets too.
+	for(var/obj/item/organ/organ in internal_organs)
+		organ.removed()
+		organ.loc = src
+
+	release_restraints(victim)
+	victim.organs -= src
+	victim.organs_by_name[limb_name] = null // Remove from owner's vars.
+
+	//Robotic limbs explode if sabotaged.
+	if(is_robotic && sabotaged)
+		victim.visible_message(
+			"<span class='danger'>\The [victim]'s [src.name] explodes violently!</span>",\
+			"<span class='danger'>Your [src.name] explodes!</span>",\
+			"<span class='danger'>You hear an explosion!</span>")
+		explosion(get_turf(owner),-1,-1,2,3)
+		var/datum/effect/effect/system/spark_spread/spark_system = new /datum/effect/effect/system/spark_spread()
+		spark_system.set_up(5, 0, victim)
+		spark_system.attach(owner)
+		spark_system.start()
+		spawn(10)
+			del(spark_system)
+		del(src)
+
+/obj/item/organ/external/proc/disfigure(var/type = "brute")
+	if (disfigured)
+		return
+	if(type == "brute")
+		owner.visible_message("\red You hear a sickening cracking sound coming from \the [owner]'s [name].",	\
+		"\red <b>Your [name] becomes a mangled mess!</b>",	\
+		"\red You hear a sickening crack.")
+	else
+		owner.visible_message("\red \The [owner]'s [name] melts away, turning into mangled mess!",	\
+		"\red <b>Your [name] melts away!</b>",	\
+		"\red You hear a sickening sizzle.")
+	disfigured = 1
+
 /****************************************************
 			   ORGAN DEFINES
 ****************************************************/
 
-/datum/organ/external/chest
-	name = "chest"
+/obj/item/organ/external/chest
+	name = "upper body"
+	limb_name = "chest"
 	icon_name = "torso"
-	display_name = "chest"
-	max_damage = 75
-	min_broken_damage = 50
+	health = 100
```

Comment:
I'd see what Zuhayr says about this, but it would probably be best to explicitly define max damage (and remove the deprecated "health" var)--unless the equation alone is meant to handle it (though on Baycode max_damage is explicitly defined).